test/unit/ObjectMakrTests/make.js

```diff
@@ -0,0 +1,38 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("object's name is John the Penguin", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            // Act
+            var madeObject = objectMaker.make(mocks.mockClassExample, {name: "John", weight: 213, age: 4});
```

Comment:
[Don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). You've written out the same object literal three times, and each time you're manually referring to the same value. That's a sign that it should be moved to a shared location.